rsocket-core/src/main/java/io/rsocket/internal/ClientServerInputMultiplexer.java

```diff
@@ -148,7 +149,7 @@ public InternalDuplexConnection(
     @Override
     public Mono<Void> send(Publisher<ByteBuf> frame) {
       if (debugEnabled) {
```

Comment:
being pedantic, shouldn't debug levels be able to change in a running app?

Reply:
I set this way so that eventually it would get JIT'd away and you could only set it startup so you don't accidentally turn on frame logging.

Reply:
Your call, it's not common.